### PR TITLE
update FA documentation

### PR DESCRIPTION
**Pull request by rgrotenbreg**

### CODE REVIEW

File: security/firely-auth/firely-auth-management-app.rst [hunk 1]
```diff
@@ -3,122 +3,18 @@
 Firely Auth User Management
 ===========================
 
-When using the SQL Server user store, you can manage the users therein with a command line utility called the 'Firely Auth Management app'. This can be used both interactively from the commandline and automated from a script or devops pipeline.
+.. note:: 
+  In Firely Auth versions < 4.0.0 there was an executable to manage users, this has been deprecated.
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:7]:
```suggestion
  In Firely Auth versions < 4.0.0 there was an executable to manage users, this has been deprecated and removed.
```

File: security/firely-auth/firely-auth-management-app.rst [hunk 2]
```diff
@@ -3,122 +3,18 @@
...
 
-.. _firely_auth_mgmt_exe:
+After you have installed, started and created your admin account, you can now manage users in the UI.
+After creation of a non-sso account, an email will be sent to the address of the account to activate the account. When this link would be expired, you can resend this email through the UI as well.
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:10]:
```suggestion
After creation of a local account (not using Single Sign-on), an email will be sent to the address of the account to activate the account. When this link expires, an admin can resend this email through the UI.
```

File: security/firely-auth/firely-auth-deploy.rst [hunk 3]
```diff
@@ -137,17 +137,23 @@ See the instructions on :ref:`running Firely Server in Docker <use_docker>` to l
...
+
+Sqlite user store
+-----------------
+
+Sqlite is setup by default and will create a database in the ./data/ folder. If you want to change this, you can alter the settings as described in :ref:`firely_auth_settings_userstore`
+
+To add users to the store, you can use the UI  or through the :ref:`firely_auth_mgmt` once the application has started.
 
-The users for the InMemory user store can be configured in :ref:`firely_auth_settings_userstore`
+When you want to use Sqlite as user store in docker, you will have to create a database file, mount it to your docker container, and adjust the Sqlite connectionstring.
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-deploy.rst:156]: Could you add a link to the database settings here?

File: security/firely-auth/firely-auth-management-app.rst [hunk 4]
```diff
@@ -3,122 +3,18 @@
...
 
-Executable
-----------
+When you want to manage the users programmatically, you can use the API's that are specified in the swagger documentation which you can find at: https://localhost:5001/swagger/ (or replace localhost with the url of your Firely Auth installation).
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:12]:
```suggestion
When you want to manage the users programmatically, you can use the API's that are specified in the swagger documentation which you can find at: https://localhost:5001/swagger/ (or replace localhost with the url of your Firely Auth installation). The API enables the creation, updating and deletion of users.
```

File: security/firely-auth/firely-auth-deploy.rst [hunk 5]
```diff
@@ -159,7 +165,7 @@ Use of the SQL Server user store requires Microsoft SQL Server version 2016 or n
 Using your favorite database administration tool:
 
 - create a new database, e.g. 'firely_auth_store'
-- in this database, execute the script ``scripts/InitializeSchema.sql``, available in the binaries
+- in this database, execute the scripts from the ``scripts/SqlServer/`` folder, available in the binaries, or let the application run the migrations by itself (but then the user must have enough privileges).
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-deploy.rst:168]: Can we summarise the permissions needed here?

File: security/firely-auth/firely-auth-management-app.rst (hunk 4 above)
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:12]: Could we add some examples here with the payload on how to call the service?

File: security/firely-auth/firely-auth-management-app.rst [hunk 6]
```diff
@@ -3,122 +3,18 @@
...
+To be able to use the api, you will need a client token (obtained trough the client_credentials flow) for a client who has the ``AllowManagementApiAccess`` property set to true.
+It is recommended to use a specific client for this API access.
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:14]: How about adding a default appsetting here which can be added via copy&paste?

File: security/firely-auth/firely-auth-management-app.rst [hunk 7]
```diff
@@ -3,122 +3,18 @@
...
-      
-      ``-u, --username``: username
-      
-      ``-cs, --connectionString``
-      
-      ``-hi, --passwordHashIterations``
-
-.. _firely_auth_mgmt_password:
-
-Password requirements
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-management-app.rst:66]: Do we have password requirements listed somewhere else?

File: security/firely-auth/firely-auth-settings.rst [hunk 8]
```diff
@@ -66,6 +66,85 @@ Nevertheless you can control the settings for Kestrel.
...
+
+Account
+^^^^^^^
+
+These settings control the account specific options:
+
+.. code-block:: json
+
+  "Account": {
+    "AuthenticationCookieExpiration": "1", // [ws][-]{ d | [d.]hh:mm[:ss[.ff]] }[ws] (provide days or timespan)
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-settings.rst:79]: We set this now to 1:30h right per default?

File: security/firely-auth/firely-auth-tutorial.rst [hunk 9]
```diff
@@ -145,7 +123,24 @@ Note that we also set the Audience in the Advanced Settings to the default value
...
 
-Step 5 - Connect Firely Server to Firely Auth
+Step 6 - Users
+^^^^^^^^^^^^^^
+
+With the required settings in place, you can start Firely Auth in PowerShell by running::
+
+    > ./Firely.Auth.Core.exe
+
+And you can access it with a browser on ``https://localhost:5001``. It will use a self-signed certificate by default, for which your browser will warn you.
```
Comment (alexzautke) [inline security/firely-auth/firely-auth-tutorial.rst:133]: I would suggest to link to mkcert and recommend creating a local cert.

File: security/firely-auth/firely-auth-management-app.rst (hunk 1 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:7]: done

File: security/firely-auth/firely-auth-management-app.rst (hunk 2 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:10]: done

File: security/firely-auth/firely-auth-deploy.rst (hunk 3 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-deploy.rst:156]: done

File: security/firely-auth/firely-auth-management-app.rst (hunk 4 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:12]: done

File: security/firely-auth/firely-auth-deploy.rst (hunk 5 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-deploy.rst:168]: done

File: security/firely-auth/firely-auth-management-app.rst (hunk 6 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:14]: done

File: security/firely-auth/firely-auth-management-app.rst (hunk 7 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:66]: in the app settings

File: security/firely-auth/firely-auth-management-app.rst [hunk 10]
```diff
@@ -3,122 +3,18 @@
...
-    ./Firely.IdentityServer.ManagementApp.exe users delete -u bob
-
-Change a user's password
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-There is no function to change a user's password directly. Instead delete the user and recreate it with the new password and any claims.
-
-.. _firely_auth_mgmt_sso_user:
-
-Creating password-less users in the UI for SSO
```
[the remark this answers is not in the document]
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:117]: yeah, API or UI

[the remark this answers is not in the document]
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-management-app.rst:117]: just mark them as SSO user

File: security/firely-auth/firely-auth-tutorial.rst (hunk 9 above)
Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-tutorial.rst:133]: for a production environment they should not use mkcert, but i will add it with a note it being for testing

Comment (rgrotenbreg) [reply security/firely-auth/firely-auth-tutorial.rst:133]: added a kestrel setup section

**Review by alexzautke — APPROVED** [review]

File: security/firely-auth/firely-auth-deploy.rst [hunk 11]
```diff
@@ -137,17 +137,23 @@ See the instructions on :ref:`running Firely Server in Docker <use_docker>` to l
...
-The InMemory user store is only meant for testing your setup or evaluating Firely Auth.
-For production use configure the SQL Server user store.
+The InMemory user store is not supported since version 4.0. You will have to set up a Sqlite or a SqlServer database and add your users in the UI or through the :ref:`firely_auth_mgmt`.
+
+.. _firely_auth_deploy_sqlite:
+
+Sqlite user store
+-----------------
+
+Sqlite is setup by default and will create a database in the ./data/ folder. If you want to change this, you can alter the settings as described in :ref:`firely_auth_settings_userstore`
```
Comment (Sieuwkje-firely) [inline security/firely-auth/firely-auth-deploy.rst]: It changed to ./Data/ in the meantime

**Review by Sieuwkje-firely — APPROVED** [review]

**Pull request merged**